doc/nrf/ug_nrf9160.rst

```diff
@@ -199,8 +201,71 @@ Likewise, in Step 7, choose the :file:`.hex` file for the application you are pr
 .. note::
    When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
```

Comment:
Not a part of the changes from this PR, but isn't this backwards?

Reply:
I think the note is correct. @gregersrygg : Could you please confirm.

Reply:
The modem and the application is updated separately. You _should_ update to the latest modem firmware if there is a new version available, but it's not a requirement.